options.go

```diff
@@ -14,10 +14,19 @@ import (
 // Idea taken from https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis
 type option func(*Context) error
 
+func DataPath() string {
+	gopath, ok := os.LookupEnv("GOPATH")
+	if !ok {
+		panic("Error locating data path, go GOPATH set")
```

Comment:
Did you try this? I think `panic` will pop a full stack trace. `log.Fatalf` may be preferable for an end-user oriented message.

Reply:
Fixed myself along with some other stuff related to logging once I got in there.